src/lib/core/components/visualizations/implementations/BarplotVisualization.tsx

```diff
@@ -398,9 +398,12 @@ function BarplotViz(props: VisualizationProps) {
               : data.value?.facets
                   .map((el: { label: string; data?: BarplotData }) => {
                     // faceted plot: here data.value is full data
-                    return el.data?.series[index].value.some(
```

Comment:
I'd suggest the following. I think it's more readable?

```
                    return el.data?.series[index]?.value.some(
```

Reply:
@bobular thank you for your reviews and tests. I agree with your suggestion so I adopted it and made commit 👍